projects/js-packages/components/components/button/style.module.scss

```diff
@@ -0,0 +1,166 @@
+.button {
+	--wp-admin-theme-color: var(--jp-black);
+	--wp-admin-theme-color-darker-10: var(--jp-black-80);
+	--wp-admin-theme-color-darker-20: var(--jp-black-80);
+	--wp-admin-border-width-focus: 1.5px;
+
+	border-radius: var(--jp-border-radius);
+
+	// Loading Mode
+	& > :global(.components-spinner) {
+		margin: 0 var(--spacing-base);
+	}
+
+	// Normal Size & Font
+	&.normal {
+		height: 40px;
```

Comment:
Should we define these values based on `--spacing-base` too. p1648820233528369-slack-C02TQF5VAJD

Reply:
7497f98